source/more-info/unsupported/home_assistant_core_version.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: "Home Assistant Core version"
+description: "More information on why Home Assistant Core version marks the installation as unsupported."
+---
+
+## The issue
+
+Supervisor considers Home Assistant Core versions older than 24 months (approximately 24 releases) as unsupported.
+We strongly recommend keeping Home Assistant Core updated to the latest version, or
+at minimum updating within 6 months (6 release cycles) to ensure you don't miss
+automatic YAML integration migrations. Migration code that converts YAML-based
+integrations to modern UI-based configuration is only available for 6 months
+after deprecation, which corresponds to 6 Home Assistant Core releases.
+
+On installations using an unsupported Home Assistant Core release, the
+Supervisor stops refreshing its update information. This means you will no
+longer receive updates for any component, including Home Assistant Core or
+Add-on updates.
```

Comment:
This is not entirely true - the update information is fetched on Supervisor restart because it's done before the unsupported issue is created.

Reply:
Uh, that is not intended to happen. For OS, we carefully made sure that the evaluation is done before fetching. I need to double check code then :cold_sweat: